### PR TITLE
[WIP] updating README/Contrib to match other sdks

### PR DESCRIPTION
**Pull request by laurafitzgerald**
## Motivation

There have been some updates to the ios-sdk README/CONRIB

## Progress

- [x] Update README/CONTRIBUTION 
- [x] Update filenames in docs/modules/ROOT/pages as per [discussion](https://groups.google.com/forum/#!topic/aerogear/pdP8BFfxcTw) on mailing list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,64 +7,31 @@
...
-1. [Troubleshooting](./docs/troubleshooting.adoc)
-1. [Service Contributor Guide](./docs/contributing-guide.adoc)
 
 ### List of SDKs
 
-AeroGear Services SDK consist of set of separate SDKs
+AeroGear Services consist of set of separate SDKs
+
+- Core - Core library used as base for other SDKs
+- Auth - Mobile Application Auth SDK (part of the core bundle)
```
Comment (danielpassos) [inline README.md]: Auth is not part of the core. It's a separated module.

File: README.md [hunk 2]
```diff
@@ -7,64 +7,31 @@
...
+- Security - Mobile Application Self Defense Checks
```
Comment (danielpassos) [inline README.md]: I know for now we have only self-defense check, but I think we should say it's a security module, not self-defense module.

Comment (laurafitzgerald) [reply README.md]: @danielpassos yes makes sense, I will update that in both prs, Android and iOS.

File: README.md (hunk 1 above)
Comment (laurafitzgerald) [reply README.md]: @danielpassos will update, thanks.

File: README.md (hunk 2 above)
Comment (laurafitzgerald) [reply README.md]: @finp based on that comment should the file structure mentioned here: https://groups.google.com/forum/#!topic/aerogear/pdP8BFfxcTw be updated to be `using-auth-module.adoc`?

Comment (finp) [reply README.md]: @laurafitzgerald  let's stick with sdk for now

**Pull request merged**